pymatgen/io/vasp/inputs.py

```diff
@@ -2280,6 +2283,43 @@ def __repr__(self) -> str:
         return f"{cls_name}({symbol=}, {functional=}, {TITEL=}, {VRHFIN=}, {n_valence_elec=:.0f})"
 
 
+def _gen_potcar_summary_stats():
+    """
+    This function solely intended to be used for PMG development to regenerate the
+    potcar_summary_stats.json.gz file used to validate POTCARs
+    """
```

Comment:
Perfect! This is great to have. Even though it's marked as private, we should add a test for this.

Reply:
Added a `Test_gen_potcar_summary_stats` to `tests/io/vasp/test_inputs.py`. Requires generating a library of fake POTCARs, which are in `tests/fake_POTCAR_library`. Code to generate this library from a real library of POTCARs is in `dev_scripts/potcar_scrambler.py`